platforms/nuttx/src/px4/microchip/mpfs/hrt/hrt.c

```diff
@@ -391,21 +503,26 @@ hrt_call_reschedule()
 		if (next->deadline <= (now + HRT_INTERVAL_MIN)) {
 			hrtinfo("pre-expired\n");
```

Comment:
Why not schedule here?

Reply:
What do you mean? We set very short interval as it has been done in stm32 side.. but do you mean even shorter interval?
Or invoke() straight away?

Reply:
sorry, got confused with the diff; this is fine